lang/java/reef-common/src/main/java/org/apache/reef/client/DriverRestartConfiguration.java

```diff
@@ -64,6 +64,12 @@
       new OptionalImpl<>();
 
   /**
+   * The amount of time the driver waits for evaluators to report back on restart. Defaults to 3 minutes. If
+   * the value is set to Integer.MAX_VALUE, the driver will wait forever.
+   */
+  public static final OptionalParameter<Integer> DRIVER_RESTART_EVALUATOR_RECOVERY_MS = new OptionalParameter<>();
```

Comment:
Please add to the documentation that this is measured in milliseconds. The suffix `MS` together with the mention of a default of "3 minutes" is confusing.